src/bootstrap/src/core/config/config.rs

```diff
@@ -1314,7 +1314,13 @@ impl Config {
 
         // Set flags.
         config.paths = std::mem::take(&mut flags.paths);
-        config.skip = flags.skip.into_iter().chain(flags.exclude).collect();
+        config.skip = flags
+            .skip
+            .into_iter()
+            .chain(flags.exclude)
+            .map(|p| t!(config.src.join(p).canonicalize()))
```

Comment:
Question: I seem to recall `canonicalize` will try to create the path if it doesn't exist, right?

Reply:
No, it'll just fail if it doesn't exist. https://doc.rust-lang.org/stable/std/fs/fn.canonicalize.html#errors

Reply:
Ah cool.